src/axom/quest/examples/shaping_driver.cpp

```diff
@@ -147,6 +148,12 @@ struct Input
       ->check(axom::CLI::NonNegativeNumber)
       ->capture_default_str();
 
+    app.add_option("-e,--percent-error", percentError)
```

Comment:
Also, can a user supply both the `percentError` parameter and the `samplesPerError` and `refinementLevel` parameters?
If the former is supplied, are the latter ignored?

Reply:
Right now, if the user supplies --percent-error and samples, then --percent-error will win. The refinement level parameter is independent since it controls the circle refinement. The IntersectionShaper::refineShape() method is set up to also explore different refinement levels. I ended up restricting it to the user-supplied refinement level because it did not affect accuracy enough (after refinement 7, at least).

Reply:
Thanks -- in that case, should we modify the command line parsing to either emit a warning, or at least mention it in the description?

Reply:
--percent-error's doc string is more explicit about dynamic refinement being used if a value is provided.